tests/xfstests/run.pm

```diff
@@ -387,10 +387,17 @@ sub umount_xfstests_dev {
     }
 }
 
+sub config_debug_option {
+    script_run('echo 1 > /proc/sys/kernel/softlockup_all_cpu_backtrace');    # on detection capture more debug information
+    script_run('echo 1 > /proc/sys/kernel/softlockup_panic');    # panic when softlockup
```

Comment:
I wonder if these actually do anything... https://github.com/os-autoinst/os-autoinst-distri-opensuse/blame/ea666f3efbf5089f3aeda401e4844c67c019d896/lib/bootloader_setup.pm#L767 we've had them for 2 years and still I haven't seen anything that useful...

Reply:
This setting helps to panic kernel and get more logs when soft lockup happens. See this difference:
Before set kernel.softlockup_panic=1: https://openqa.suse.de/tests/7924858#step/run/1125
After set kernel.softlockup_panic=1: https://openqa.suse.de/tests/7964813#step/run/1137

Reply:
@SeroSun  is it possible that: `softlockup_all_cpu_backtrace` is the one actually providing all this useful output? 

Because both systems have kernel.softlockup_panic=1 enabled:

Before set kernel.softlockup_panic=1: https://openqa.suse.de/tests/7924858/logfile?filename=serial0.txt
```
Command line: BOOT_IMAGE=/boot/vmlinuz-5.14.21-150400.3-default root=UUID=4c81affb-ac5c-4810-bf3c-e1d4c63666c7 splash=silent video=1024x768 plymouth.ignore-serial-consoles console=ttyS0 console=tty kernel.softlockup_panic=1 resume=/dev/disk/by-uuid/2a708a10-7955-40a5-b922-91c99f037404 mitigations=auto/
```

 After set kernel.softlockup_panic=1: https://openqa.suse.de/tests/7964813/logfile?filename=serial0.txt
```
[    0.000000][    T0] Command line: BOOT_IMAGE=/boot/vmlinuz-5.14.21-150400.3-default root=UUID=4c81affb-ac5c-4810-bf3c-e1d4c63666c7 splash=silent video=1024x768 plymouth.ignore-serial-consoles console=ttyS0 console=tty kernel.softlockup_panic=1 resume=/dev/disk/by-uuid/2a708a10-7955-40a5-b922-91c99f037404 mitigations=auto
```

Reply:
@foursixnine `softlockup_all_cpu_backtrace` works in multi-processor VM, e.g. 64bit-smp. with this setting system could print more log when softlockup happen(my verify run is in 64bit, so not show how it works). This setting also need to enable `kernel.softlockup_panic`, otherwise no output at all in openqa when softlockup.